authors/apps/authentication/serializers.py

```diff
@@ -244,6 +244,22 @@ class ResetPasswordSerializer(serializers.ModelSerializer):
         write_only=True
     )
 
+    def validate_password(self, password):
+        """
+            validates that  password is longer than 8 characters
+            password is alphanumeric
+        """
+        if len(password) < 8:
+            raise serializers.ValidationError(
+                "Password should atleast be 8 characters.")
+        if not re.search(r'[0-9]', password) or not re.search(r'[a-zA-Z]', password) or not re.search(r'[!?@#$%^&*.]', password):
```

Comment:
Please adopt the team conventions as per the engineering book. Probably use flake8 to lint your code.

Reply:
@mwinel If it is the length you are referring to, I have fixed it. If anything else, please elaborate